framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -378,15 +378,15 @@ int MediaPlayerImpl::getVolume()
 	return ret;
 }
 
-void MediaPlayerImpl::getVolumePlayer(int &ret)
+void MediaPlayerImpl::getVolumePlayer(uint8_t &ret)
 {
 	medvdbg("MediaPlayer Worker : getVolume\n");
-
-	ret = get_output_audio_volume();
+	//ToDo: Add error handling code
+	get_output_audio_volume(&ret);
```

Comment:
you return below case in get_output_audio_volume.
```
	if (volume == NULL) {
		return AUDIO_MANAGER_INVALID_PARAM;
	}
```
previous is right.

Reply:
The error handling code and new enum value of player_result_t are added.